spring/spring/pom.xml

```diff
@@ -237,6 +237,43 @@
                     </instructions>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
this turns out to be unnecessary after changing xsd version to 8.2

I will remove changes in pom.xml.